src/views/tutorial-view/components/featured-in-collections/index.tsx

```diff
@@ -1,46 +1,43 @@
-import { getCollectionSlug } from 'views/collection-view/helpers'
-import CardLink from 'components/card-link'
-import { Collection as ClientCollection } from 'lib/learn-client/types'
-
-interface FeaturedInCollectionsProps {
-  collections: CollectionCardProps[]
-}
-
-// @TODO move this to the collection card component file when we make it
-export interface CollectionCardProps
-  extends Pick<
-    ClientCollection,
-    'id' | 'name' | 'slug' | 'theme' | 'description'
-  > {
-  numTutorials: number
-}
+import CollectionCard from 'views/tutorial-view/components/collection-card'
+import { FeaturedInCollectionsProps } from './types'
+import s from './featured-in-collections.module.css'
 
 // This should render the eventual `CollectionCard` component (doesn't exist yet)
 // which will be used on many other views
 export function FeaturedInCollections({
+  className,
```

Comment:
Passing `className` to the root element allows us to set spacing styles on the component, to space it relatively to adjacent components at the page level.